### ISSUE TITLE
Setting default firewall rule

### ISSUE THREAD

**Issue by robinAwallace** [UpCloudLtd/terraform-provider-upcloud#272]
Allow setting the default firewall rule from terraform, that you can do via Upcloud hub.

**Comment by ka-myl** [comment]
Hi @robinAwallace, this is already in progress. Not sure about the final shape of it, but there will be option to do that via TF soon :)

**Comment by ka-myl** [comment]
Hi @robinAwallace this should now be available in v2.5.0. You can simply do it by specifying last rule(s) like this:

```
firewall_rule {
    action    = "drop"
    direction = "in"
}

firewall_rule {
    action = "drop"
    direction = "out"
}
```

I'll close this issue, let us know if you bump into any other problems with this.
